### DISCUSSION TITLE
How to add  an embedded YouTube video

### DISCUSSION THREAD

**Discussion by bahitosV2V** [thorsten/phpMyFAQ#2635] [Q&A]
How to add an embedded YouTube video phpMyFAQ 3.1.13

**Comment by thorsten** [comment]
You can simply paste the HTML snippet from youtube.com into the FAQ entry, when you switch in the editor to the source code view.

**Comment by c1972** [comment]
And if you always wanted to display **the most recent video from a specific YouTube-Channel**, then use my following solution:

First get the **Channel-ID**, either from the YT source code or, for example, with this: https://seostudio.tools/youtube-channel-id

The Channel-ID always starts with **UC**. We change that and make **UU** from it.

The link then should look like this:
`https://www.youtube.com/embed?listType=playlist&amp;list=UU0w4AA42ItXQEb9aZld87-w`

In phpMyFAQ, click on **<>** for the code view, then enter the following:

`<div class="videoWrapper"><iframe src="https://www.youtube.com/embed?listType=playlist&amp;list=UUhere-the-remaining-ID"></iframe></div>`

This will display the most recent video from the YT-Channel of your choice. They post a new one? ...already on your site! (Cool thing?)

Let's assume you also want to display the 3 previous videos (perhaps a little smaller below?) that's also very easy.

To the ID from the first example, simply add `&amp;index=2` for the second-to-last video. Or a 3 for the one before, etc.

A link for older Videos could look like this:
`https://www.youtube.com/embed?listType=playlist&amp;list=UU0w4AA42ItXQEb9aZld87-w&amp;index=2`

With a bit of CSS you can display the whole thing in full width:

```
.videoWrapper {
position: relative;
padding-bottom: 56.25%; /* 16:9 */
height: 0;
}
.videoWrapper iframe {
position: absolute;
top: 0;
left: 0;
width: 100%;
height: 100%;
}
```
And that's it already. Amazing or? Hope it helps! :) Greetz Chris
